pkg/api/vlabs/types.go

```diff
@@ -195,10 +195,12 @@ type AgentPoolProfile struct {
 	DiskSizesGB         []int  `json:"diskSizesGB,omitempty"`
 	VnetSubnetID        string `json:"vnetSubnetID,omitempty"`
 	IPAddressCount      int    `json:"ipAddressCount,omitempty"`
-
+	OsImageOffer      	string `json:"osImageOffer,omitempty"`
```

Comment:
@amanohar  Also looking at how it is organised currently, we should ideally leave the OS choice at agent/master pool level instead of putting it in Linux Profile (one per cluster definition :-(). At agent level we have a choice to mix the OS For Example, CoreOS and Ubuntu or (in future) different version of Windows RS1 and RS3 in same cluster.